src/Mobile/Controls/Player.xaml.cs

```diff
@@ -51,8 +51,10 @@ private void UpdatePlayPause()
         epiosdeTitle.Text = this.playerService.CurrentEpisode.Title;
         authorText.Text = $"{this.playerService.CurrentShow?.Author} - {this.playerService.CurrentEpisode?.Published.ToString("MMM, d yyy")}";
 
+#if WINDOWS || MACCATALYST
```

Comment:
Could the problem with maccatalyst be the fact that we are specifying `macOS` as the platform in:
https://github.com/microsoft/dotnet-podcasts/blob/df5d09c15825c761f394766917eae06b0e45e7b4/src/Mobile/Controls/Player.xaml#L45C10-L45C10
and not `MacCatalyst`